pkg/sql/schema_changer.go

```diff
@@ -958,6 +960,54 @@ func (sc *SchemaChanger) done(ctx context.Context) (*sqlbase.ImmutableTableDescr
 					})
 				}
 			}
+
+			//** Computed Col Swap **//
+			if computedColumnSwap := mutation.GetComputedColumnSwap(); computedColumnSwap != nil {
```

Comment:
this code seems duplicated with the code that starts a job for the primary key swap case. Can you pull that code out and reuse it here?